src/rpc/blockchain.cpp

```diff
@@ -1704,6 +1729,9 @@ UniValue scantxoutset(const JSONRPCRequest& request)
             "   }\n"
             "   ,...], \n"
             " \"total_amount\" : x.xxx,          (numeric) The total amount of all found unspent outputs in " + CURRENCY_UNIT + "\n"
+            " \"rawsweep_tx\" : \"value\",       (string) The hex-encoded raw transaction of the optional sweep transaction\n"
+            " \"rawsweep_vsize\" : \"value\",     (numeric) virtual transaction size of the sweep transaction including signatures\n"
+            " \"rawsweep_fee\" : \"value\",       (numeric) Estimated fee for the sweep transaction in " + CURRENCY_UNIT + "\n"
```

Comment:
Nit: "Estimated fee" implies that the fee is unknown, which is not true given that the exact fee can be calculated.